auth-service/src/main/resources/templates/thymeleaf/register-user-form.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html lang="en" class="govuk-template " xmlns:th="http://www.thymeleaf.org">
+
+<head th:replace="~{fragments/template.html :: head (title='DELTA | Register')}">
+    <title>DELTA | Register</title>
+</head>
+
+<body class="govuk-template__body govuk-body">
+<a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
+<header th:replace="~{fragments/template.html :: header}"></header>
+
+<!--/*@thymesVar id="deltaUrl" type="java.lang.String" */-->
+<!-- List of errors paired with error links -->
+<!--/*@thymesVar id="allErrors" type="java.lang.ArrayList" */-->
+<!--/*@thymesVar id="firstNameErrorMessages" type="java.lang.ArrayList" */-->
+<!--/*@thymesVar id="firstName" type="java.lang.String" */-->
+<!--/*@thymesVar id="lastNameErrorMessages" type="java.lang.ArrayList" */-->
+<!--/*@thymesVar id="lastName" type="java.lang.String" */-->
+<!--/*@thymesVar id="emailAddressErrorMessages" type="java.lang.ArrayList" */-->
+<!--/*@thymesVar id="emailAddress" type="java.lang.String" */-->
+<!--/*@thymesVar id="confirmEmailErrorMessages" type="java.lang.ArrayList" */-->
+<!--/*@thymesVar id="confirmEmailAddress" type="java.lang.String" */-->
+
+<div class="govuk-width-container" id="registerPage">
+    <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item"><a class="govuk-breadcrumbs__link" data-th-href="${deltaUrl}">Home</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">Register</li>
+    </ol>
```

Comment:
@BenRamchandani is this ok as is or should I look into the CSS? How it looks on new register page vs how it look on a page on delta
![image](https://github.com/communitiesuk/delta-auth-service/assets/114008899/4c389306-e04c-4b55-80b3-da2a702670e3)
![image](https://github.com/communitiesuk/delta-auth-service/assets/114008899/a3e7704b-e4bf-47cf-843b-ac15cbcefbd8)

Reply:
Also the extra space between Gov.uk and Delta?

Reply:
I think that's fine